pkgs/by-name/bo/bootcommander/package.nix

```diff
@@ -0,0 +1,45 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  libopenblt,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "bootcommander";
+  version = "1.15.00";
```

Comment:
title says 1.7.00. also currently the latest tag is [openblt_v011901](https://github.com/feaser/openblt/releases/tag/openblt_v011901)